fish-pepper/jolokia/install.dck

```diff
@@ -5,11 +5,13 @@
   var jolokiaArtifact = jolokiaVersion.match(/^2.*$/) ? "jolokia-agent-jvm" : "jolokia-jvm";
   var mavenRepo = fp.blockOpts.mavenRepo || "http://central.maven.org/maven2";
 }}# Jolokia agent
-RUN mkdir -p /opt/jolokia \
+RUN mkdir -p /opt/jolokia/etc \
  && curl {{= mavenRepo }}/org/jolokia/{{= jolokiaArtifact }}/{{= jolokiaVersion }}/{{= jolokiaArtifact }}-{{= jolokiaVersion }}-agent.jar \
          -o /opt/jolokia/jolokia.jar
 ADD jolokia-opts {{= destPath }}
 RUN chmod 444 /opt/jolokia/jolokia.jar \
```

Comment:
Found the reason again: To play nicely with run-java-sh  the non-s2i base image copies this script  to `/opt/run-java-options` which will be [automatically included by run-java.sh](https://github.com/rhuss/run-java-sh/blob/717153c13623332ed39a48acac9805ecf5e12bfd/fish-pepper/run-java-sh/fp-files/run-java.sh#L117-L126) if it exists. That way jolokia-opts and run-java.sh are nicely decoupled (only the user of the template  connects them both together by providing this destination as template argument)

For s2i its directly sourced in `/usr/local/s2i/run` from the default `/opt/jolokia-opts` , which makes sense so that the environment variables are also available for custom `/deployments/bin/run.sh` (run-java.sh is not necessarily called.

So it makes some sense to have this as paramter.